### PR TITLE
Documentation for home-assistant/home-assistant-polymer#3505

### PR DESCRIPTION
**Pull request by thomasloven**
**Description:**

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant-polymer#3505

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/10200"></a>

### CODE REVIEW

File: source/_lovelace/markdown.markdown [hunk 1]
```diff
@@ -36,3 +36,25 @@ content: >
...
+  - light.kitchen_lights
+card:
+  type: markdown
+  content: |
+    The lights that are on are:
+    {% for l in config.entities %}
+      - {{ l.entity }}
+    {%- endfor %}
+
+    And the door is {% if is_state('binary_sensor.door', 'on' %} open {% else %} closed {% endif %}.
```
Comment (balloob) [inline source/_lovelace/markdown.markdown:70]: You need to put `{% raw %}` tags around your templates. 

Also, the rest of the docs don't mention at all that templates are allowed ??

Comment (thomasloven) [reply source/_lovelace/markdown.markdown:70]: Uh... yes it does? What's going on here?
It's in the code on the next branch: https://github.com/home-assistant/home-assistant.io/blame/next/source/_lovelace/markdown.markdown#L21
but not shown in the diff?

Comment (thomasloven) [reply source/_lovelace/markdown.markdown:70]: Fixed.
I have no idea why git got that wrong...

**Review by frenck — APPROVED** [review]
LGTM!

**Pull request merged**